docs/examples/intranet_agents.md

```diff
@@ -11,55 +12,70 @@ provides an answer.
 
 ## Description
 
-Provide the top-level "front-man" agent a question and it will call down-chain agents to determine if they are responsible for all or part of question.
-It will then ask those down-chain agents what they need in order to handle their part of the inquiry. Once the requirements are gathered, the top agent
-delegates the inquiry and the fulfilled requirements to the appropriate down-chain agents. Once all down-chain agents respond, the top agent will compile
-their responses and return the final response.
+Provide the top-level "front-man" agent a question and it will call down-chain agents to determine if they are responsible
+for all or part of question. It will then ask those down-chain agents what they need in order to handle their part of the
+inquiry. Once the requirements are gathered, the top agent delegates the inquiry and the fulfilled requirements to the
+appropriate down-chain agents. Once all down-chain agents respond, the top agent will compile their responses and return
+the final response.
 
-The "front-man" agent can call IT, Finance, Procurement, Legal, and HR agents. Those agents, in turn, can call other, more specific agents. For example,
-the HR agent can call Benefits or Payroll agents. The agent network has the shape of a tree that is two layers deep.
+The "front-man" agent can call IT, Finance, Procurement, Legal, and HR agents. Those agents, in turn, can call other,
+more specific agents. For example, the HR agent can call Benefits or Payroll agents. The agent network has the shape of
+a tree that is two layers deep.
 
 ---
 
 ## Example Conversation
 
-### Human:
-```
+### Human
+
+```text
 I recently got married
 ```
 
-### AI (intranet_agents):
-```
+### AI (intranet_agents)
+
+```text
```

Comment:
This is actually markdown output by an agent and should NOT be reformatted

Reply:
Changed to markdown